src/libraries/System.Net.Quic/src/System/Net/Quic/QuicConnectionOptions.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace System.Net.Quic
+{
+    internal class QuicConnectionOptions
+    {
+        public int BidirectionalStreamCount { get; set; } = 100;
+        public int UnidirectionalStreamCount { get; set; } = 100;
```

Comment:
Why do you default to this? I recommend a default of 0.

Reply:
I understand MsQuic defaulting to 0 for stream counts as it follows the QUIC spec, but I'd prefer it work for the 99% case and set it to a default that allows streams. People can set it to 0 if they don't want to create streams in certain directions. I'm not sure if 100 is a great number, I just wanted it to be greater than 0.

The other option is to make it a constructor parameter to session or connection to require people to set a stream count. However, that doesn't scale if we'd like to expose more options.